### DISCUSSION TITLE
XDM.msi - corrupted or newer version already installed!

### DISCUSSION THREAD

**Discussion by XDM-Dude22** [subhra74/xdm#955] [Q&A]
Hi Forum:

A few days ago I uninstalled XDM because, for whatever reason, it had completely stopped downloading software, audio and video files.

When trying to reinstall it I kept getting one of two error messages. For xdmsetup.msi "A newer or same version of the software is already installed." and for xdm-setup.msi "The configuration data for this product is corrupt. Contact your support personnel". There definitely is _no_ newer version of the XDM installed and xdm-setup.msi can't be corrupted, either. An friend of mine had no issue installing both versions on Win 10. Btw, I'm still using Win7 x64 and am mostly very happy with it.

As for uninstalling XDM I used Revo Uninstaller Pro. In the Registry I then deleted everything that was named XDM, XDMAN or Xtreme Download Manager. Reboots didn't help, I've also googled and 'duckduckgoed' extensively for a solution, without any luck.

I'd be very grateful if you could help me fix this problem! I'm sorry If I forgot to include any additional info relevant to the issue.

Thanks so much,

XDM-Dude22 😁

**Comment by subhra74** [comment]
Please try latest version 8.0.18, and let me know if this solves the issue. 
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta
